.pipelines/templates/e2e-tests.yml

```diff
@@ -22,6 +32,12 @@ steps:
     - script: |
           set -eux  # fail on error
           source env/bin/activate
-          pytest
+          if [ -z ${TEST_SUITE} ]; then pytest -v; else pytest -v -m ${TEST_SUITE}; fi
       workingDirectory: e2e-tests
+      env:
+        ${{ if ne(parameters.analyzer_base_url, '') }}:
```

Comment:
What will happen if it's empty? Should it continue? Perhaps add a trace message

Reply:
In most cases it will empty and expected, and the env var will not be set - for the tests to get the default urls (localhost for running against containers)